docs/openapi/components/schemas/common/USDAPPQ391SpecimensForDetermination.yml

```diff
@@ -0,0 +1,243 @@
+$linkedData:
+  term: USDAPPQ309APestInterceptionRecord
+  '@id': https://w3id.org/traceability#USDAPPQ309APestInterceptionRecord
+title: USDA PPQ 309A Pest Interception Record
+description: A record describing a pest sample submitted to a USDA APHIS (Animal and Plant Health Inspection Service) PPQ (Plant Protection and Quarantine) office as well as the findings of the recipient lab.
+type: object
+properties:
+  type:
+    type: array
+    readOnly: true
+    const:
+      - USDAPPQ309APestInterceptionRecord
+    default:
+      - USDAPPQ309APestInterceptionRecord
+    items:
+      type: string
+      enum:
+        - USDAPPQ309APestInterceptionRecord
+  priority:
+    title: Priority
+    description: "The priority of the determination. Must be one of: Urgent, Prompt, Routine."
```

Comment:
Why are some `description` values wrapped in double-quotes (like this one), and others not (like the next one)? It seems that either all should or none should...

Reply:
Quotes allow characters considered special in yml, like colons